src/pages/preparing-for-tech-talk-part-1-motivation/index.zh-hans.md

```diff
@@ -0,0 +1,81 @@
+---
+title: '做好技术分享，第一章：驱动'
+date: '2018-12-26'
+spoiler: 这是我对美妙分享灵感的秘方
+---
+
+我做过一些我认为非常不错的 [技](https://www.youtube.com/watch?v=xsSnOQynTHs) [术](https://www.youtube.com/watch?v=nLF0n9SACd4) [分享](https://www.youtube.com/watch?v=dpw9EHDh2bM)。
+
+有时人们问我如何准备好一个分享。对于每个人来说，答案都是不同的，下面我分享的是对于我有用的部分。
+
+**本文是这个系列分享的第一部分**，本文中我阐述了如何准备一个技术分享--从产生一个想法到现场分享。
+
+* **做好技术分享，第一章：驱动**
+* **[做好技术分享，第一章：内容，原因和方法](/preparing-for-tech-talk-part-2-what-why-and-how/)**
+* 未完待续
+
+<p />
+
+---
+
+**本文中，我只会关注第一步：我是为什么并且如何挑选话题的。**这并不会提供很多实践建议，但是可以让你关注一些正确的点。
+
+---
+
+是什么*激励*你，让你来做技术分享？
+
+可能分享就是你现在工作的一部分，可能你希望以此在公司中获得更多认同，来获得一份更好的工作，或者晋升。可能你仅仅是希望大家更了解你的兴趣或工作项目。
+
+我们称这些驱动是*外在*的，它们关系到别人如何评价你以及你的工作。但如果你已经拥有了你想要的声望和金钱，你仍然会做一个技术分享吗？为什么？
+
+也许你享受传道受业；也许你享受学习，而做分享正是一个追根究底的好方式；也许你希望引起一个话题的讨论；也可能你只想对一个观点衡短论长。
+
+这些*内在*驱动并不只是代表希望获得专业认可，这些东西对你来说有固有的价值。不同的人有不同的内在驱动，了解你自己的内在驱动是有意义的，你可以通过他们追本溯源，更清楚地认识自己。
+
+例如，以下是我的内在驱动：
+
+* **我乐于分享灵感性的想法。**有时，一个想法能改变我的思考方式，能打开新世界的大门。大部分情况下，一个人思考是闭门造车，我希望其他人来和我交流，能让我茅塞顿开。对我来说，一次分享，是一个集思广益的过程。（在我年轻时，虽然我的音乐品味并不突出，但我喜欢制作磁带。而现在，我却喜欢分享，生活总是会带给你惊喜。）
+
+* **我乐于以简单的方式解释事物。** 当我理解一个想法时，我会获得比吃甜食更大的愉悦感。学习对我来说不是易事，所以当我搞清楚某事时，我希望能通过我的方式点醒被它困扰的人。我尝试记住在我恍然大悟之前我的观点是如何的，这样能帮助别人在接受我的分享时获得同样的过程。（我小时候经常让别人问我问题，这让人难以忍受，而做分享则是一个更有成效的让别人提问的方式。）
+
+---
+
+把这两个内在驱动结合起来，我就有了一个让我满意的分享的秘方：**以简单的方式解释一个灵感性的想法。**
+
+---
+
+这是我的准则，一千个人眼中就有一千个哈姆雷特，好好想想什么分享让你感到特别，他们之间有何形式上的类似吗？（本系列接下来会更多地讨论分享形式）
+
+![Luna Lovegood invoking a Patronus Charm. Image © 2007 Warner Bros. Ent](./patronus.jpg)
+
+做一个和你驱动一致的分享对你在以下方面是有帮助的：
+
+1. **话题变得更容易挑选。**我的准则是「解释一个灵感性的想法，以及你为何需要关注它」。在我学会的有趣的概念上，通过实施这一准则我可以搞定我的分享提案。只要持续地和聪明人交流，听取他们值得发扬光大的想法，我就总可以有一些值得讨论的话题。而你可以挖掘更多可能的准则！
+
+2. **讲台上你不会觉得胆怯。**在分享前的三十秒，我会觉得恐惧，不过一旦我开始演讲，这就是我的舞台了。分享一个灵感性想法的驱动力战胜了被批评或者做错事的恐惧。（当然，以上都是只有你做好了准备才有效，我们会在接下来的文章中讨论）
+
+3. **你会变得更让人信服。** [Sophie](https://mobile.twitter.com/sophiebits/status/1077723835481284608) 在这方面表达得比我好：如果你对一个话题充满了热忱，观众也会被你感染，也会对此感兴趣。激情并不只代表挥着手或者大声说话，即使你很冷静，人们同样可以感受到背后的信念。（这就是为何我们就算听不懂[歌词](https://www.youtube.com/watch?v=6SWIwW9mg8s)，也可以感受一首歌）
+
+---
+
+当你对某个话题感到非常兴奋时，还有一个原因会激励你--你会感觉自己是宏伟事业的一部分，这会让人充满信心。
+
+其实我的分享是关于一个想法，并不是关于我，而我只是一个信使。那些上万的观众，他们评判的不是我（即使他们是如此认为），他们只是来听取我的想法。我的只是把想法传递给其他人。把这些想法植入内心后，来自分享的精神压力就会消失。
+
+---
+
+寻找契合你驱动的准则可以帮助你确定你的话语。但是，你如何找到可以应用这个准则的特定话题呢？
+
+**根据我的经验，良好的谈话始于对话。**有人向我解释了一个想法，然后我试着向别人解释，我和十几个人讨论它，最后我能找到「普适」的解释。想法有时会被忽视或误解，我会试图让不同人以不同的眼光看待它。 
+
+对我来说，分享只是一种概括这些对话并使它们成为一对多而非一对一的方式。它就像一个“库”，是你从许多面对面和社交媒体讨论的「应用程序代码」中提取出来。 
```

Comment:
```suggestion
对我来说，分享只是一种概括这些对话并使它们成为一对多而非一对一的方式。它就像一个「库」，是你从许多面对面和社交媒体讨论的「应用程序代码」中提取出来。 
```
If you use square brackets at one place to replace quotation marks, use it consistently.